src/components/Counter.js

```diff
@@ -51,9 +55,14 @@ const mapStateToProps = (state) => {
     };
 };
 
+const mapDispatchToProps = dispatch => ({
```

Comment:
This works! There is also a shorthand for this, where you don't have to use `mapDispatchToProps` at all, and you can simply put the name of the action in braces, `{ increment, decrement }` and put that in `connect` as the second argument. Then you would just use the action's name throughout the component (so, `this.props.increment` instead of `this.props.incrementHandler`)

So `connect` would look like this:
```
connect(mapStateToProps, { increment, decrement })(Counter)
```

Your code is not wrong or bad, this is just an alternate option!